drivers/hmis/app/models/hmis/hud/client_access_loader.rb

```diff
@@ -14,17 +14,16 @@ def fetch(items)
     client_ids = items.map { |i| i.first.id }.compact.uniq
 
     group_view_t = Hmis::GroupViewableEntity.arel_table
-    client_project_t = Hmis::Hud::ClientProject.arel_table
     orphan_client_ids = Hmis::Hud::Client.
-      left_outer_joins(:client_projects).
+      left_outer_joins(:enrollments).
       where(id: client_ids).
-      where(client_project_t[:project_id].eq(nil)).
+      where(arel.e_t[:project_id].eq(nil)).
```

Comment:
I don't think this one is correct. `orphan_client_ids` should represent "unenrolled clients", as the comment below indicates ("client is not associated with a project").

We want clients that have NO enrollments, which I believe is:
```suggestion
      where(arel.e_t[:id].eq(nil)).
```

We should probably add a test to verify this behavior (if this didn't cause a failure) but feel free to bump that to a new issue if its getting to be too much scope-creep.

Reply:
Great catch! This doesn't seem to be covered by current tests. Actually I'm not quite sure where to add a test for this as the root level client resolver uses viewable_by which catches this before the access loader. Maybe we need unit tests for the access loaders